sys-auth/elogind/elogind-255.5-r1.ebuild

```
@@ -172,6 +172,22 @@ pkg_postinst() {
 			elog "configuration remember to migrate those to new configuration file."
 		fi
 	done
+
+	local file files
+	# find custom hooks excluding known (nvidia-drivers, sys-power/tlp)
+	if [[ -d "${EROOT}"/$(get_libdir)/elogind/system-sleep ]]; then
+		readarray -t files < <(find "${EROOT}"/$(get_libdir)/elogind/system-sleep/ \
+			-type f \( -not -iname ".keep_dir" -a \
+				-not -iname "nvidia" -a \
+				-not -iname "49-tlp-sleep" \)) || die
```

`|| die` is not super important given it doesn't abort the package being installed (in postinst it's just for attention), but this currently checks if `readarray` failed rather than `find` and is no-op
```suggestion
				-not -iname "49-tlp-sleep" \) || die)
```
(on a side-note, this wouldn't work in normal bash code given exit doesn't work there, but portage can specially die inside `< <()`)